.travis.yml

```diff
@@ -4,31 +4,16 @@ node_js:
   - "0.10"
   - "4"
   - "5"
-env:
-  - FIREFOX_VERSION=nightly
-#  - FIREFOX_VERSION=aurora
-#  - FIREFOX_VERSION=beta
-#  - FIREFOX_VERSION=release
-
+addons:
+  firefox: latest-beta
 notifications:
   irc: "irc.mozilla.org#jpm"
-
 before_install:
-  - "export DISPLAY=:99.0"
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
```

Comment:
I don't know what this old command was for but it seemed deliberate. Any specific reason for simplifying it?